requirements/tests

```diff
@@ -1,19 +1,14 @@
 coverage >= 4.1
-# TODO(vytas): Our use of testtools breaks under pytest 8.2 along the lines of
-#   https://github.com/pytest-dev/pytest/issues/12263, unpin when fixed
-#   (or drop support for testtools altogether?)
-pytest >= 7.0, < 8.2
+pytest >= 7.0
 pyyaml
 requests
-# TODO(vytas): Check if testtools still brings anything to the table, and
-#   re-enable if/when unittest2 is adjusted to support CPython 3.10.
-testtools; python_version < '3.10'
+testtools
```

Comment:
do we still need it?

Reply:
Yes, we still support `testtools`, we haven't even deprecated that support yet.